packages/connect-explorer/src/pages/settings.mdx

```diff
@@ -71,6 +72,11 @@ export const Settings = () => {
                     Init error: {initError}
                 </ErrorMessage>
             )}
+            {isInitSuccess && (
+                <ConfirmationMessage data-test="@settings/init-success">
+                    Init success!
+                </ConfirmationMessage>
+            )}
```

Comment:
we still don't set it for web version, am I right?

Reply:
It does show on web. On webextension you get both messages in case of sucessful init and proxy handshake